README.md

```diff
@@ -1 +1,13 @@
+# Chef Jump-In Site
+
+A simple website to list Github issues labelled with `Type: Jump In`.
```

Comment:
Should be specific that this is geared towards chef repos since they're hard coded in.